Replace repository content with ROS2 packages

Cyano0 opened the pull request:
<!--
     For Work In Progress Pull Requests, please use the Draft PR feature,
     see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

-->

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update

## Description

Replace repository content with ROS2 packages


## Related Tickets & Documents
A minimal version of SENTOR in ROS2, need to be tested. Could be used as a starting point.

<!--
For pull requests that relate or close an issue, please include them
below.  We like to follow [Github's guidance on linking issues to pull requests](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

For example having the text: "closes #1234" would connect the current pull
request to issue 1234.  And when we merge the pull request, Github will
automatically close the issue.
-->

- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes._

## [optional] Are there any post deployment tasks we need to perform?

ibrahimhroob commented:
Thanks @Cyano0 for this PR, can we please solve the conflict with the main branch?

Cyano0 commented:
> Thanks @Cyano0 for this PR, can we please solve the conflict with the main branch?

I don't know how to solve the conflict -- I got messages like this if I want to pull something to ros2:
Your branch is up to date with 'origin/master'.
CONFLICT (modify/delete): .gitignore deleted in e20ec4a (Replace repository content with ROS2 packages) and modified in HEAD.  Version HEAD of .gitignore left in tree.
CONFLICT (rename/rename): msg/Monitor.msg renamed to src/msg/Monitor.msg in HEAD and to sentor_msgs/msg/Monitor.msg in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/rename): msg/SentorEvent.msg renamed to src/msg/SentorEvent.msg in HEAD and to sentor_msgs/msg/SentorEvent.msg in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/rename): msg/TopicMap.msg renamed to src/msg/TopicMap.msg in HEAD and to sentor_msgs/msg/TopicMap.msg in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): CHANGELOG.rst renamed to src/CHANGELOG.rst in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): config/execute.yaml renamed to src/config/execute.yaml in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): config/map.yaml renamed to src/config/map.yaml in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): config/rob_lindsey.yaml renamed to src/config/rob_lindsey.yaml in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): config/test.yaml renamed to src/config/test.yaml in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): launch/sentor.launch renamed to src/launch/sentor.launch in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): launch/topic_mapping.launch renamed to src/launch/topic_mapping.launch in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): msg/MonitorArray.msg renamed to src/msg/MonitorArray.msg in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): msg/TopicMapArray.msg renamed to src/msg/TopicMapArray.msg in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): scripts/test.py renamed to src/scripts/test.py in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): scripts/topic_mapping_node.py renamed to src/scripts/topic_mapping_node.py in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): setup.py renamed to src/setup.py in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): src/sentor/CustomLambdaExample.py renamed to src/src/sentor/CustomLambdaExample.py in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): src/sentor/CustomProcessExample.py renamed to src/src/sentor/CustomProcessExample.py in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): src/sentor/ROSTopicHz.py renamed to src/src/sentor/ROSTopicHz.py in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (modify/delete): src/src/sentor/ROSTopicHz.py deleted in e20ec4a (Replace repository content with ROS2 packages) and modified in HEAD.  Version HEAD of src/src/sentor/ROSTopicHz.py left in tree.
CONFLICT (rename/delete): src/sentor/TopicMapServer.py renamed to src/src/sentor/TopicMapServer.py in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/delete): src/sentor/TopicMapper.py renamed to src/src/sentor/TopicMapper.py in HEAD, but deleted in e20ec4a (Replace repository content with ROS2 packages).
CONFLICT (rename/rename): srv/GetTopicMaps.srv renamed to src/srv/GetTopicMaps.srv in HEAD and to sentor_msgs/srv/GetTopicMaps.srv in e20ec4a (Replace repository content with ROS2 packages).
error: could not apply e20ec4a... Replace repository content with ROS2 packages

Cyano0 commented:
Closed as I created a new request.
